Riot/Modules/Authentication/AuthenticationViewController.m

```diff
@@ -1124,6 +1141,18 @@ - (void)showResourceLimitExceededError:(NSDictionary *)errorDict
     }];
 }
 
+#pragma mark - UITextFieldDelegate
+
+- (void)textFieldDidBeginEditing:(UITextField *)textField
+{
+    CGRect frameInScrollView = [textField convertRect:textField.frame toView:self.authenticationScrollView];
+    //  add an offset to also see some below of the textField
+    frameInScrollView.origin.y += 20;
+    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
This is clearly a hack that seems to work and enables to wait the keyboard animation finish. We will need to update `KeyboardAvoider` to expose methods and properties and remove this.

Reply:
Exposed `func scroll(to view: UIView, animated: Bool = true)` method in `KeyboardAvoider` and called.